meta-store/src/meta_store.rs

```diff
@@ -61,19 +67,22 @@ pub struct SplitMetaData {
     generation: usize,
 }
 
+/// A split manifest entry.
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct ManifestEntry {
+    /// The file name.
     file_name: String,
+    /// File siize in bytes.
```

Comment:
typo: size...

Also, the checksum is missing.

But I think we don't want the manifest entry in the meta store anymore so it does not matter I guess.